packages/datadog-plugin-mongodb-core/src/index.js

```diff
@@ -1,14 +1,29 @@
 'use strict'
 
 const DatabasePlugin = require('../../dd-trace/src/plugins/database')
+const coalesce = require('koalas')
 
 class MongodbCorePlugin extends DatabasePlugin {
   static get id () { return 'mongodb-core' }
   static get component () { return 'mongodb' }
   // avoid using db.name for peer.service since it includes the collection name
   // should be removed if one day this will be fixed
   static get peerServicePrecursors () { return [] }
+
+  configure (config) {
+    super.configure(config)
+    this.config.mongoHeartbeatDisabled = coalesce(
+      config.mongoHeartbeatDisabled,
+      process.env.DD_TRACE_MONGO_HEARTBEAT_DISABLED,
```

Comment:
`DD_TRACE_MONGO_HEARTBEAT_DISABLED` should be `DD_TRACE_MONGODB_HEARTBEAT_DISABLED` to match the integration name. While this is not important for now, we might be able to automate this in the future if a standard is respected.

Reply:
done

Reply:
and makes sense